app/services/github/fetch_all_repos.rb

```diff
@@ -0,0 +1,10 @@
+class Github::FetchAllRepos
+  def initialize(user_name, oauth_token)
+    @user_name = user_name
+    @github_client = Utilities::GithubApi::Client.new(oauth_token)
+  end
+
+  def call
+    @github_client.fetch_all_repositories(@user_name)
```

Comment:
Instead of keep on using instance variable directly, access using accessors

```
class Github::FetchAllRepos
  attr_accessor :user_name, :github_client
  def initialize(user_name, oauth_token)
    @user_name = user_name
    @github_client = Utilities::GithubApi::Client.new(oauth_token)
  end

  def call
    github_client.fetch_all_repositories(user_name)
  end
end
```